docs/src/rust-client/delegated_proving_tutorial.md

```diff
@@ -0,0 +1,195 @@
+# Delegated Proving
+
+_Using delegated proving to minimize transaction proving times on computationally constrained devices_
+
+## Overview
+
+In this tutorial we will cover how to use delegated proving with the Miden Rust client to minimize the time it takes to generate a valid transaction proof.
+
+## Prerequisites
+
+This tutorial assumes you have basic familiarity with the Miden Rust client.
+
+## What we'll cover
+
+- Explaining what "delegated proving" is and its pros and cons
+- How to use delegated proving with the Rust client
+
+## What is Delegated Proving?
+
+Before diving into our code example, let's clarify what in the world "delegated proving" actually is.
+
+Delegated proving is the process of outsourcing a part of the ZK proof generation of your transaction to a third party. For certain computationally constrained devices such as mobile phones and web browser environments, generating ZK proofs might take too long to ensure an acceptable user experience. Devices that do not have the computational resources to generate Miden proofs in under 1-2 seconds can use delegated proving to provide a more responsive user experience.
+
+_How does it work?_ When a user choses to use delegated proving, they send off a portion of the zk proof of their transaction to a dedicated server. This dedicated server generates the remainder of the ZK proof of the transaction and submits it to the network. Submitting a transaction with delegated proving is trustless, meaning if the delegated prover is malicious, the could not compromise the security of the account that is submitting a transaction to be processed by the delegated prover. The only downside of using delegated proving is that it reduces the privacy of the account that uses delegated proving, because the delegated prover would have knowledge of the transaction that is being proven.
```

Comment:
nit: I think there is a type in "meaning if the delegated prover is malicious, **the** could not compromise the security ..."

Reply:
I didn't check, but what does that mean? "they send off a portion of the zk proof of their transaction to a dedicated server." 

What do we send? It should be the full zk proof that is generated on the server, or am I mistaken?